internal/app/machined/pkg/controllers/network/resolver_config.go

```diff
@@ -225,6 +225,10 @@ func (ctrl *ResolverConfigController) parseMachineConfiguration(logger *zap.Logg
 		return spec, false
 	}
 
+	if len(resolvers) > 0 {
+		spec.DNSServers = make([]netip.Addr, 0, len(resolvers))
```

Comment:
this change seems to be irrelevant, or AI slop

Reply:
You should probably start with an issue and a way to reproduce the problem, but this code change doesn't make any sense to me.